dashboard/test/factories/factories.rb

```diff
@@ -573,6 +573,12 @@
       login_type {'email'}
       grades {['pl']}
     end
+
+    trait :old_section do
```

Comment:
Do we really need a trait for it? Why don't just do it in the test?
```ruby
create :section, created_at: 1.year.ago
```

Reply:
> Do we really need a trait for it? Why don't just do it in the test?
> 
> ```ruby
> create :section, created_at: 1.year.ago
> ```


Ah good call, I tried updating the created_at field after the create and it didn't work, but I somehow didn't think to just override created_at during the creation 🤦